src/core/vector/qgsvectorlayer.h

```diff
@@ -2834,7 +2868,7 @@ class CORE_EXPORT QgsVectorLayer : public QgsMapLayer, public QgsExpressionConte
     QgsVectorLayer( const QgsVectorLayer &rhs );
 #endif
     //! Returns the minimum or maximum value
-    QVariant minimumOrMaximumValue( int index, bool minimum ) const;
+    void minimumOrMaximumValue( int index, QVariant &minimum, QVariant &maximum, bool calculateMinimum, bool calculateMaximum ) const;
```

Comment:
what about using pointers or a flag so the call is a bit more readable?

Reply:
Or return a `QgsRange` ?